src/Compilers/VisualBasic/VbcCore/project.lock.json

```diff
@@ -1,1896 +0,0 @@
-{
```

Comment:
FWIW, I noticed today that the project.lock.json file for VbcCore kept changing on my whenever I created a branch off master and rebuilt. Something may be incorrect here.